skimage/restoration/deconvolution.py

```diff
@@ -8,62 +8,63 @@
 
 
 def wiener(image, psf, balance, reg=None, is_real=True, clip=True):
-    r"""Wiener-Hunt deconvolution
+    r"""Restore image using Wiener-Hunt deconvolution.
 
-    Return the deconvolution with a Wiener-Hunt approach (i.e. with
-    Fourier diagonalisation).
+    Wiener-Hunt deconvolution is a restoration method which follows a Bayesian
+    approach [1]_.
 
     Parameters
     ----------
-    image : ndarray
-       Input degraded image (can be n-dimensional).
+    image : ([P, ]M, N) ndarray
+       Degraded image.
     psf : ndarray
-       Point Spread Function. This is assumed to be the impulse
-       response (input image space) if the data-type is real, or the
-       transfer function (Fourier space) if the data-type is
-       complex. There is no constraints on the shape of the impulse
+       Point spread function (PSF). Assumed to be the impulse
+       response (input image space) if the data type is real, or the
+       transfer function (Fourier or frequency space) if the data type is
+       complex. There is no constraint on the shape of the impulse
        response. The transfer function must be of shape
-       `(N1, N2, ..., ND)` if `is_real is True`,
-       `(N1, N2, ..., ND // 2 + 1)` otherwise (see `np.fft.rfftn`).
+       ``(N1, N2, ..., ND)`` if `is_real is True`,
```

Comment:
I'm actually not sure what these shapes are... What are `N1, ..., ND`? I can see that this notation was introduced in b8ce4d0c2c65684b131870abfe60228136108d4e by @m-albert and @rfezzani: Can you please help me here?

Reply:
It's been a bit since I worked on this.

Before https://github.com/scikit-image/scikit-image/commit/b8ce4d0c2c65684b131870abfe60228136108d4e, which generalized the function from 2D to ND, the docstring was:
```
The transfer function must be of shape `(M, N)` if
       `is_real is True`, `(M, N // 2 + 1)` otherwise (see
       `np.fft.rfftn`).
```
The new docstring adapted this to the n-dimensional case so that `N1, ...., ND` refers to the n-dimensional shape of the input image. So in 2D (N1, N2) would be equivalent to (M, N).

Sorry for stating the obvious though 😅